api/database/models/User.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      /* Test.hasMany(models.User,{
+        foreignKey:'roleId'
+      }) */
+    }
+  };
+  User.init({
+    FullName: DataTypes.STRING,
+    email: DataTypes.TEXT,
+    password: DataTypes.STRING,
+    phone: DataTypes.INTEGER
```

Comment:
change datatype integer to string because for the length of the phone it is not enough